visualization/.eslintrc.js

```diff
@@ -4,7 +4,13 @@ module.exports = {
 		"es6": true,
 		"node": true
 	},
-	"extends": ["prettier"], // prettier must be last in array to override other configs
+	"extends": [
+		"eslint:recommended",
+		"plugin:@typescript-eslint/eslint-recommended",
+		"plugin:@typescript-eslint/recommended",
+		"prettier",
+		"prettier/@typescript-eslint",
```

Comment:
is this working? as the comment says, prettier needs to be last in the array

Reply:
Yes, those are both part of prettier.